# This service is used to command the state of a physics simulation, e.g. stop, start, restart.

auv_physics/SimulationInstruction command
---
auv_physics/SimulationState state